### PR TITLE
tus 1.0 Compat

### PR DESCRIPTION
**Pull request by MMasterson**

### CODE REVIEW

**Comment by MMasterson** [comment]
@kvz @Acconut

File: Pod/Classes/TUSResumableUpload.m [hunk 1]
```diff
@@ -164,9 +190,9 @@ - (void)connection:(NSURLConnection *)connection
     
     switch([self state]) {
         case CheckingFile: {
-            if (httpResponse.statusCode != 200) {
-                NSLog(@"Server responded with %d. Restarting upload",
-                      httpResponse.statusCode);
+            if ([httpResponse statusCode] != 200) {
```
Comment (Acconut) [inline Pod/Classes/TUSResumableUpload.m]: `204 No Content` would also be an acceptable status code.

**Comment by Acconut** [comment]
Thanks for your work but there are still some points missing in order to claim this 1.0-compatible:
* The `Offset` header has been renamed to `Upload-Offset`.
* The same is for the length, `Final-Length` is now `Upload-Length`.
* The `Tus-Resumable` header must be included in every request by the client.

**Comment by MMasterson** [comment]
@Acconut - got it, give me a day or so

**Pull request closed without merging**